### ISSUE TITLE
Wrong cursor on text presents 

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#1331]
## Bug Description

While interacting with text presents, the present does not act like button. The cursor does not change on hover. The element is not taggable and there is no hover and clicked ( tapped ) state. 

## Expected Behaviour

A button tag should be used and styled, so this element acts like a user would expect. 

## Screenshot
![Screenshot from 2020-04-24 14-47-22](https://user-images.githubusercontent.com/237508/80220086-40686f80-863b-11ea-87f6-1577534628e1.png)

**Comment by swissspidy** [comment]
Duplicate of #1104?

**Comment by spacedmonkey** [comment]
@swissspidy Yes!
